Makefile

```diff
@@ -4,6 +4,9 @@
 ROOT_DIR=${CURDIR}
 LLVM_PROJ_DIR?=$(ROOT_DIR)/src/llvm-project
 
+# Setup ninja verbose option at the very beginning for easily comment it out
+NINJA_FLAGS:=$(NINJA_FLAGS) -v
```

Comment:
How about we set `NINJA_FLAGS=-v` in the CI scripts and then we can avoid having this as the default at all?

Reply:
OK with that